docs/BUILD.md

```diff
@@ -45,15 +45,15 @@ cmake --install build
 </details>
 
 <details>
-<summary>AL2</summary>
+<summary>AL2 and AL2023</summary>
 
 ```bash
 # install build dependencies
-sudo yum groupinstall "Development Tools"
-sudo yum install cmake3
+sudo yum groupinstall -y "Development Tools"
+sudo yum install -y cmake3
```

Comment:
I'm not sure we should have "-y" in our default build instructions. The customer would need it for an automated script, but for a manual build (especially if it's their first time running it / they're unsure what they're doing) that doesn't seem right.